api/v1alpha1/dspipeline_types.go

```diff
@@ -69,11 +69,24 @@ type APIServer struct {
 	// Include sample pipelines with the deployment of this DSP API Server. Default: true
 	// +kubebuilder:default:=false
 	// +kubebuilder:validation:Optional
-	EnableSamplePipeline bool   `json:"enableSamplePipeline"`
-	ArgoLauncherImage    string `json:"argoLauncherImage,omitempty"`
-	ArgoDriverImage      string `json:"argoDriverImage,omitempty"`
+	EnableSamplePipeline bool `json:"enableSamplePipeline"`
```

Comment:
Just curious: Why changing v1alpha1 API since v1 already had the changes?

Reply:
because under the hood v1alpha1 gets converted to v1, so their spec must matches, we should just never be adding things to v1alpha1 that are not in v1